pkg/util/pod/util.go

```diff
@@ -682,6 +693,10 @@ func CanPodBePreempted(pod *v1.Pod) int {
 	return -1
 }
 
+func GetPodMetaName(pod *v1.Pod) string {
```

Comment:
several functions are doing the same thing, don't add a new one

Reply:
fixed, use `podutil.GetPodKey` instead